src/Sentry.Unity.Editor/SentryPreprocessBuild.cs

```diff
@@ -10,32 +14,39 @@ public class SentryPreprocessBuild : IPreprocessBuildWithReport
 
         public void OnPreprocessBuild(BuildReport report)
         {
-            // var options = AssetDatabase.LoadAssetAtPath<SentryCliOptions>(SentryWindows.SentryCliOptionsAssetPath);
-
             switch (report.summary.platform)
             {
-                case BuildTarget.StandaloneOSX:
-                case BuildTarget.StandaloneLinux64:
-                case BuildTarget.StandaloneWindows:
-                case BuildTarget.StandaloneWindows64:
-                case BuildTarget.tvOS:
                 case BuildTarget.iOS:
-                case BuildTarget.XboxOne: // pdbs?
-                    // platformGroup = Standalone
-                    // sentry-cli upload-dif
-                    break;
-                case BuildTarget.WebGL:
-                    // sentry-cli sourcemaps
-                    break;
-                case BuildTarget.Android:
-                    // sentry-cli proguard
+                    CheckIOSEditorImportSettingsOnWindows(Path.Combine("Packages", SentryPackageInfo.GetName(), "Editor", "iOS",
+                        "Sentry.Unity.Editor.iOS", ".meta"));
                     break;
                 default:
                     // nothing to do
                     break;
             }
-            // file: summary.outputPath
-            // guid: summary.guid
+        }
+
+        internal void CheckIOSEditorImportSettingsOnWindows(string metaFilePath, IApplication? application = null)
+        {
+            application ??= ApplicationAdapter.Instance;
+            if (application.Platform != RuntimePlatform.WindowsEditor)
+            {
+                return;
+            }
+
+            if (!File.Exists(metaFilePath))
+            {
+                return;
+            }
+
+            var meta = File.ReadAllText(metaFilePath);
+            if (meta.Contains("OS: OSX"))
+            {
+                new UnityLogger(new SentryUnityOptions()).LogWarning("You're trying to build for iOS on " +
```

Comment:
The idea of a link to the docs was a good one.
Maybe a sub page on Native Support just about this?